src/app/storage/components/AddEditStorageModal/AddEditStorageForm.tsx

```diff
@@ -96,6 +98,20 @@ const InnerAddEditStorageForm = ({ values, touched, errors, ...props }) => {
           <FormErrorDiv id="feedback-bucket-name">{errors.bucketName}</FormErrorDiv>
         )}
       </FormGroup>
+      <FormGroup label="S3 URL" fieldId={s3UrlKey}>
```

Comment:
Actually, thoughts on changing this to `S3 Endpoint`? Seems to be the common language I see across services using the S3 API.

Reply:
No problem

Reply:
Updated